### PR TITLE
Add dropdown button for AWS region selection in cluster creation

### PR DESCRIPTION
**Pull request by tuommaki**
There is now an upcoming support for inter-region clusters in aws-operator and
to use that one needs to be able to specify the region the cluster should be
created in. Therefore add a dropdown selection for this region.

### CODE REVIEW

File: src/components/create_cluster/index.js [hunk 1]
```diff
@@ -197,6 +200,31 @@ class CreateCluster extends React.Component {
...
+          'eu-west-2',
+          'eu-west-3',
+          'eu-north-1',
+          'sa-east-1',
+          'us-gov-east-1',
+          'us-gov-west-1',
+      ];
+      return regions;
+  }
+
```
Comment (marians) [inline src/components/create_cluster/index.js]: Will this move to the info endpoint?

File: src/components/create_cluster/release_selector.js [hunk 2]
```diff
@@ -250,8 +250,8 @@ function mapStateToProps(state) {
   var provider = state.app.info.general.provider;
   var releases = Object.assign({}, state.entities.releases.items);
 
-  var activeSortedReleases = _.filter(releases, release => release.active);
-  activeSortedReleases = activeSortedReleases.map(release => release.version);
+  //var activeSortedReleases = _.filter(releases, release => release.active);
+  var activeSortedReleases = _.map(releases, release => release.version); //activeSortedReleases.map(release => release.version);
```
Comment (tuommaki) [inline src/components/create_cluster/release_selector.js]: :exclamation: REVERT!

File: src/components/create_cluster/index.js (hunk 1 above)
Comment (tuommaki) [reply src/components/create_cluster/index.js]: Yes, I did couple shortcuts here and there in order to get this running E2E within two days :slightly_smiling_face:

**Comment by tuommaki** [comment]
I guess this PR can act as a base, but proper version needs more work.

**Comment by marians** [comment]
@tuommaki What should we do with this PR? AFAIK the companions in api etc. have been closed. Should we close this one, too? We can dig it up here once we want to implement this.

**Comment by tuommaki** [comment]
Yes, let's close this one and redo when needed.

**Pull request closed without merging**